spec/requests/v1/workflow/steps_spec.rb

```diff
@@ -12,6 +12,7 @@
 
   before do
     sign_in(user)
+    SSJ::TeamMember.create(person: person, ssj_team: ssj_team, role: SSJ::TeamMember::PARTNER, status: SSJ::TeamMember::ACTIVE)
   end
```

Comment:
curious what happened here, not obvious why.   prob useful to comment in these situations as we'll all forget in 6 mo

Reply:
If you look above, a person is being created with an `ssj_team`.  But when you call `person.reload.ssj_team` it returns `nil` because an `SSJ::TeamMember` record was never created. I'll add a comment.